packages/builder/src/index.html

```diff
@@ -6,6 +6,7 @@
 
 	<title>Budibase Builder</title>
 
+	<link href="https://cdn.jsdelivr.net/npm/remixicon@2.3.0/fonts/remixicon.css" rel="stylesheet">
```

Comment:
There appears to be a bug with using the remix NPM module and svelte together - just pulling it from CDN for now.

Reply:
Do we source a better icon set?

Reply:
I actually quite like remix - it's our build process that just needs a bit of TLC